File: Instructor Java/Week03 Java/src/main/java/case02/NumberRunner.java
package case02;
/**
 * C02NumberRunner
 */
public class NumberRunner implements Runnable{

    public void run() {
        var sample = new Number();
        sample.arithmeticOperators();
        sample.numericDataType();
        sample.demoInt(456,23);
        sample.demoDouble(456.44,7.22);
        sample.demoMix(456,7.55);
        sample.covertFromString("2.3");
    }
}

File: Instructor Python/Week02/src/case02/NumberRunner.py
from .Number import Number
class NumberRunner():
    def run(self) :
        sample =Number()
        sample.arithmeticOperators()
        sample.numericDataType()
        sample.demoInt(456,23)
        sample.demoFloat(456.44,7.22)
        sample.demoComplex((3+4j),(5+1j))
        sample.demoMix(456,7.55)
        sample.covertFromString("2.3")